doc/rst/developer/bestPractices/GeneratedCode.rst

```diff
@@ -1,18 +1,19 @@
-=================================
+	=================================
 All About Compiler-Generated Code
 =================================
 
 Where is it? How to compile it?
 -------------------------------
 
-By default: in a temporary directory,
-deleted when the compiler exits (only the executable remains).
+By default: in a temporary directory, deleted when the compiler exits (only the executable remains).
+
+:``--savec DIRECTORY``: makes it go to DIRECTORY and stay there.
```

Comment:
The spacing on this renders a little odd:

![pic1](https://user-images.githubusercontent.com/3595449/54138588-29929980-43f6-11e9-8e9a-6e8071e377a7.png)

I think it could be improved.